app/javascript/util/routeUtil.ts

```diff
@@ -29,7 +29,7 @@ const getLocalizedPath = (
 }
 
 const localizedPathGetter = (newPathNonLocalized: string) => (
-  currentPath: string | undefined
+  currentPath: string | undefined = window.location.pathname
```

Comment:
I just added this default so this code is still testable but we don't need to pass the pathname in explicitly everywhere.